Loop/Managers/LoopAppManager.swift

```diff
@@ -368,6 +368,47 @@ class LoopAppManager: NSObject {
         }
         return false
     }
+    
+    private func resetLoop() {
+        resetLoopUserDefaults()
```

Comment:
It seems here is where the pump manager could be warned of deactivation. That interface is already in LoopKit. Willing to add it?

Reply:
Okay, it's added :)